src/mnemonic.rs

```diff
@@ -220,10 +225,12 @@ impl Mnemonic {
     }
 
     /// Consume the `Mnemonic` and return the phrase as a `String`.
-    ///
-    /// This operation doesn't perform any allocations.
-    pub fn into_phrase(self) -> String {
-        self.phrase
+    pub fn into_phrase(mut self) -> String {
+        // Create an empty string and swap values with the mnemonic's phrase.
+        // This allows `Mnemonic` to implement `Drop`, while still returning the phrase.
+        let mut phrase = String::new();
+        mem::swap(&mut self.phrase, &mut phrase);
```

Comment:
```suggestion
        mem::swap(&mut self.phrase, String:new());
```

Reply:
I borked this suggestion, just woke up. [Fixed on master](https://github.com/maciejhirsz/tiny-bip39/commit/7d807b02fb6710ad42e0934956665f7ba0d3b80d).